kubernetes_deploy/production/deployment.yml

```diff
@@ -37,7 +37,7 @@ spec:
           name: http
         env:
         - name: ALLOWED_HOSTS
-          value: ".laa-fala-production.apps.live-1.cloud-platform.service.justice.gov.uk .find-legal-advice.justice.gov.uk"
+          value: ".laa-fala-production.apps.live.cloud-platform.service.justice.gov.uk .find-legal-advice.justice.gov.uk"
```

Comment:
I think this needs to stay as is as those live-1 domains still work on the new cluster

Reply:
ok, does it also need to reference live or just live-1? And what about in the ingress files?